stores/post_store.jsx

```diff
@@ -67,9 +67,24 @@ class PostStoreClass extends EventEmitter {
         this.removeListener(POST_PINNED_CHANGE_EVENT, callback);
     }
 
-    getLatestPostId(channelId) {
-        const postsInChannel = getState().entities.posts.postsInChannel[channelId] || [];
-        return postsInChannel[0];
+    getLatestPostId(channelId, allowSystemMessages) {
```

Comment:
can we make this one a selector in `mattermost-redux` instead? is really handy to have